datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/agent_reqs.py

```diff
@@ -15,15 +16,19 @@
     context_settings=CONTEXT_SETTINGS,
     short_help="Verify that the checks versions are in sync with the requirements-agent-release.txt file",
 )
-def agent_reqs():
+@click.argument('check', autocompletion=complete_valid_checks, required=False)
+def agent_reqs(check):
```

Comment:
Minor nit, and applies to all commands. The three words `check`, `checks`, and `checks_option` are used interchangeably. I think this made it a bit harder for me to reason through on first look, would it make sense to standardize these to `checks_option` or `checks` for everything?

Reply:
Can see the confusion, but did apply `check` consistently as the argument name to all validations.  

The odd bit comes from how that singular arg can expand to many checks via keywords `all` or `changed`, but in both cases we are still expecting a singular input argument.

Reply:
Ahh ok that makes sense then. I had thought you could pass a list of checks into these calls. If its singular then 👍